unit tests fails if file /tmp/tt exists

Steps to reproduce:

``` sh
$ echo 'test' > /tmp/tt
$ mage unit            
Running unit tests...
fatal: No names found, cannot describe anything.
?   	github.com/tarantool/tt/cli	[no test files]
?   	github.com/tarantool/tt/cli/cmd	[no test files]
?   	github.com/tarantool/tt/cli/cmdcontext	[no test files]
?   	github.com/tarantool/tt/cli/codegen	[no test files]
--- FAIL: TestConfigureCli (0.00s)
    configure_test.go:50: 
        	Error Trace:	configure_test.go:50
        	Error:      	Expected nil, but got: 0x8
        	Test:       	TestConfigureCli
    configure_test.go:72: 
        	Error Trace:	configure_test.go:72
        	Error:      	Expected nil, but got: 0x8
        	Test:       	TestConfigureCli
FAIL
FAIL	github.com/tarantool/tt/cli/configure	0.002s
?   	github.com/tarantool/tt/cli/modules	[no test files]
?   	github.com/tarantool/tt/cli/rocks	[no test files]
ok  	github.com/tarantool/tt/cli/running	(cached)
ok  	github.com/tarantool/tt/cli/ttlog	(cached)
?   	github.com/tarantool/tt/cli/util	[no test files]
?   	github.com/tarantool/tt/cli/version	[no test files]
FAIL
Error: running "go test ./cli/..." failed with exit code 1
```

--- a/cli/configure/configure_test.go
+++ b/cli/configure/configure_test.go
@@ -12,6 +12,13 @@ import (
 	"github.com/tarantool/tt/cli/util"
 )
 
+// cleanupTempDir cleanups temp directory after test.
+func cleanupTempDir(tempDir string) {
+	if _, err := os.Stat(tempDir); !os.IsNotExist(err) {
+		os.RemoveAll(tempDir)
+	}
+}
+
 // Test Tarantool CLI configuration (system and local).
 func TestConfigureCli(t *testing.T) {
 	assert := assert.New(t)
@@ -30,12 +37,15 @@ func TestConfigureCli(t *testing.T) {
 	// we run `go test`, which compiles the functions again.
 	assert.Equal(cmdCtx.Cli.ConfigPath, "tarantool.yaml")
 
+	testDir, err := ioutil.TempDir(os.TempDir(), "tarantool_tt_")
+	t.Cleanup(func() { cleanupTempDir(testDir) })
+	assert.Nil(err)
 	// Test local cofniguration.
 	cmdCtx.Cli.IsSystem = false
-	cmdCtx.Cli.LocalLaunchDir = os.TempDir()
+	cmdCtx.Cli.LocalLaunchDir = testDir
 	cmdCtx.Cli.ConfigPath = ""
 
-	expectedConfigPath, err := util.JoinAbspath(os.TempDir(), "tarantool.yaml")
+	expectedConfigPath, err := util.JoinAbspath(testDir, "tarantool.yaml")
 	assert.Nil(err)
 	ioutil.WriteFile(expectedConfigPath, []byte("config-file"), 0755)
 
@@ -54,7 +64,7 @@ func TestConfigureCli(t *testing.T) {
 	// Test default configuration (no flags specified).
 	cmdCtx.Cli.LocalLaunchDir = ""
 	cmdCtx.Cli.ConfigPath = ""
-	dir, err := ioutil.TempDir(os.TempDir(), "temp")
+	dir, err := ioutil.TempDir(testDir, "temp")
 	assert.Nil(err)
 
 	// Check if it will go down to the bottom of the directory looking

